freeipa/map.jinja

```diff
@@ -47,6 +47,8 @@ RedHat:
   ldap_conf: /etc/openldap/ldap.conf
   sssd_conf: /etc/sssd/sssd.conf
   named_conf: /etc/named.conf
+  admin:
+    password: {{ salt['pillar.get']('freeipa:server:admin:password', pillar.freeipa.server.ldap.password) }}
```

Comment:
In general, I don't like the idea of requiring the password to be on pillar.
Can we make it "if admin:password" is defined?

Reply:
I feel it should be otherwise - pillar.freeipa.server.ldap.password should if not define should get the admin password from freeipa:server:admin:password.